jfix-zookeeper/src/test/kotlin/ru/fix/zookeeper/lock/PersistentExpiringLockManagerTest.kt

```diff
@@ -66,6 +68,51 @@ internal class PersistentExpiringLockManagerTest {
         callbackFailed.get().shouldBeFalse()
     }
 
+    @Test
+    fun `concurrency releasing and scheduled prolonging lock`() {
+        val manager = createLockManager(PersistentExpiringLockManagerConfig(
+                lockCheckAndProlongInterval = Duration.ofMillis(10)
+        ))
+        val lockIds = mutableListOf<LockIdentity>()
+        for (i in 0 until 100) {
+            val lockId = createLockIdentity()
+            manager.tryAcquire(lockId) { }
+            lockIds.add(lockId)
+        }
+
+        val doneThreadCounter = AtomicInteger(0)
+        val releaseExceptions = mutableListOf<Exception>()
+        val executor = Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors())
```

Comment:
Runtime.getRuntime().availableProcessors() behavior could be strange, especially during launch in containerized environment. Not sure that number of cpu 1 will be ok for you.  Use explicit number instead.